app/models/user.rb

```diff
@@ -6,4 +6,7 @@ class User < ApplicationRecord
   def most_recent_posts
     Post.where(AuthorId: id).last(3)
```

Comment:
- [ ] Here is a great place to solve the `n + 1` query problem. We can replace the `where` with `includes`, so our code will look similar to this `Post.includes(AuthorId: id).last(3)`. Let us kindly replicate this in the methods similar to this

Reply:
Thank you very much for your valuable opinion. I am correcting that.

Reply:
Done.

Reply:
well resolved! ✔️